docs/LOAD-BALANCER.md

```diff
@@ -0,0 +1,29 @@
+Documentation for Load-Balancer:
+
+## Server Setup Instructions:
+* [Setup Nginx](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04) for Ubuntu 18.04 on a new EC2 instance. *Supplementary step*: Add [Nginx](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/) PPA to run latest stable Nginx version, instead of the default OS Nginx, since it's not updated as frequently and can lag on some feature/security fixes.
+* Setup LetsEncrypt according to [this](https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-18-04) guide on the server.
+* The Elastic IP assigned to the load-balancer instance _must_ be: `54.172.90.245`.
+
+## Server Administration:
+* How to login to the Nginx Load-balancer server: 
+	* Login to bastion server: `ssh aws-admin` (if you have the SSH config setup), or `ssh ec2-user@jump.dosomething.org`.
+	* `ssh nginx-lb`. In case the `/etc/hosts` file is borked, you can also use `ssh 10.100.60.26`. 
+* All Nginx server config files live in `/etc/nginx/conf.d` and take the filname format: `example-com.conf`.
+* To check Nginx config: `sudo nginx -T`. 
```

Comment:
`filname` should be `filename`

Reply:
Fix is [here](https://github.com/DoSomething/infrastructure/pull/240).